sklearn/tests/test_pipeline.py

```diff
@@ -227,6 +227,7 @@ def test_pipeline_score_params():
     assert_equal(pipe.score(X), 3)
     assert_equal(pipe.score(X, y=None), 3)
     assert_equal(pipe.score(X, sample_weight=np.array([2, 3])), 8)
+    assert_raises(TypeError, pipe.score, X, foo='bar')
```

Comment:
I think testing for an explicit error message would be better... (`assert_raise_message`)